src/diff/amfGraphDeltaTypes.ts

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import * as AmfGraphTypes from "./amfGraphTypes";
+
+/**
+ * External property, added by jsondiffpath to indicate that the node property is an array
+ */
+export const JSON_DIFF_PATCH_ARRAY_KEY = "_t";
+export const JSON_DIFF_PATCH_ARRAY_VALUE = "a";
+
```

Comment:
```suggestion
export type ArrayDiffMarker = { [JSON_DIFF_PATCH_ARRAY_KEY]: typeof JSON_DIFF_PATCH_ARRAY_VALUE };
```
This object is reused a few times, giving it a name makes it more clear what it's there for.

Reply:
instead of this I used typeof JSON_DIFF_PATCH_ARRAY_VALUE